argo-workflows/remedieate-cve-bump-ui-dependencies.patch

```diff
@@ -1,7 +1,9 @@
 From d64bec425f46a6418f37a617f6ea63924f830f17 Mon Sep 17 00:00:00 2001
 From: Batuhan Apaydin <batuhan.apaydin@chainguard.dev>
 Date: Thu, 21 Nov 2024 14:54:20 +0300
-Subject: [PATCH] upgrade deps
+Subject: [PATCH] Bump Node Dependencies in argo-workflows UI
+
+Remedieate(CVE): GHSA-67mh-4wv8-2f99(upgrade-esbuild-to-0.25.0), GHSA-grv7-fg5c-xmjg(upgrade-braces-to-3.0.3)
```

Comment:
This mentions upgrading `braces` but that isn't included in the patch: is that intentional?